app/tasks/migrate.php

```diff
@@ -60,6 +63,13 @@
 
         while (!empty($projects)) {
             foreach ($projects as $project) {
+                /**
+                 * Skip user projects with id 'console'
+                 */
+                if ($project->getId() === 'console' && $project->getInternalId() !== 'console') {
```

Comment:
Is this getInternalId or getId?

Reply:
It's intended 👍🏻 

Some time ago you were able to create projects with the id `console` and they didn't work. This is just making sure to not migrate them.